src/Geometry.cpp

```diff
@@ -82,6 +85,15 @@ double Geometry::getWidth() {
 
 
 /**
+ * @brief Returns the total depth (z extent) of the Geometry in cm.
+ * @return the total depth of the Geometry (cm)
+ */
+double Geometry::getDepth() {
```

Comment:
I completely agree - the height, width, and depth can be a bit confusing since they don't explicitly refer to a dimension. Unless there is any objections, I'm going to go with the notation `Geometry::getWidthX()`, `Geometry::getWidthY()`, `Geometry::getWidthZ()` since that is the notation used in the `Lattice` object in OpenMOC and OpenCG.